modes.go

```diff
@@ -259,7 +259,7 @@ func RunTest(resultChannel chan Result, workload WorkloadGenerator, rateLimiter
 	start := time.Now()
 	partialStart := start
 	iter := NewTestIterator(workload)
-	for !iter.IsDone() {
+	for !iter.IsDone() && atomic.LoadUint32(&stopAll) == 0 {
```

Comment:
This is already checked in `IsDone()`.
See https://github.com/scylladb/scylla-bench/blob/e27d33b520b19ae45a0de74e92c5a955ba02bb3d/modes.go#L242.

Reply:
Oh, you're right. Apparently there's other reason why scylla-bench does not terminate due to CTRL+C. I send this patch basing [on this diff](https://github.com/scylladb/scylla-bench/compare/0a5e6a9dacf8234a45b24a392b8ab289e78e133b...master#diff-b7df09dd46922678810a1554511bb537L234).